### DISCUSSION TITLE
the file shown in links

### DISCUSSION THREAD

**Discussion by lmilbaum** [community/community#43560] [Other Feature Feedback, Questions, & Ideas] [answered]
Question

Section 3 in the following [doc](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) is a bit unclear.
What does the following statement mean ?

> If a repository contains more than one CONTRIBUTING file, then **_the file shown in links_** is chosen from locations in the following order: the .github directory, then the repository's root directory, and finally the docs directory.

**Comment by auchynnikau** [comment, marked as answer]
This statement means that if a repository contains multiple files with the name `contributing.md` or `CONTRIBUTING.md`, GitHub will look for these files in a specific order, and the first file that it finds will be displayed in a link on the `https://github.com/{username}/{repo-name}/contribute` page right after the "Helping" heading.

The order in which GitHub looks these files is as follows:

1. `.github/contributing.md`
2. `.github/CONTRIBUTING.md`
3. `CONTRIBUTING.md`
4. `docs/contributing.md`
5. `docs/CONTRIBUTING.md`

So Github will first look for files in the `.github` directory, then in the root directory, and finally in the `docs` directory. For example, if a repository contains both `.github/contributing.md` and `CONTRIBUTING.md`, the link on the repository homepage will point to `.github/contributing.md` because it is the first file in the above list.

**Reply by lmilbaum** [reply]
Thank @auchynnikau for your prompt reply. Could you please show me where the "Help" heading located? I can't seem to find it.

**Reply by auchynnikau** [reply]
Sorry for missing out, from the homepage of the repository go to the `https://github.com/{username}/{repo-name}/contribute` page and here you will see the helping heading an right after that the contribution link mentioned above.

<img width="1400" alt="image" src="https://user-images.githubusercontent.com/33329898/211161197-491e4e26-ce8c-4530-a501-e6d5a18e7ec7.png">

**Reply by lmilbaum** [reply]
Apologies. The following links returns 404. What am I missing?

https://github.com/renovatebot/renovate/contribution
https://github.com/python-gitlab/python-gitlab/contribution
https://github.com/github/docs/contribution

**Reply by auchynnikau** [reply]
`contribution` => `contribute`

**Reply by lmilbaum** [reply]
yay... working now. Thank you so much.
